includes/fields/class-evf-field-captcha.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Captcha field
+ *
+ * @package EverestForms\Fields
+ * @since   1.6.5
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * EVF_Field_Captcha Class.
+ */
+class EVF_Field_Captcha extends EVF_Form_Fields {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->name   = esc_html__( 'Captcha', 'everest-forms' );
+		$this->type   = 'captcha';
+		$this->icon   = 'evf-icon evf-icon-captcha';
+		$this->order  = 130;
```

Comment:
@coderkoala wasn't it 160?

Reply:
Ahh... I looked up country's order, and it is at 120, so I guessed 130 would be the next apparent order. Should I bump it up to 160?

Reply:
Since it's not in the pro field so it has some intension I guess. Also, make sure if the PRO is activated only but not specifically required addon then we have to show a popup.

Reply:
@shivapoudel I changed the order to reflect the same as Pro's field on 8a3e478.

As for the tests, I did 3 sets, EVF with pro, EVF without pro, and with/without Captcha Addon.

Each time, the message was consistent when clicked on the disabled field:

![image](https://user-images.githubusercontent.com/30156167/78971843-2d7a7900-7b2c-11ea-879e-8a490a575d4c.png)